shell/common/crash_keys.cc

```diff
@@ -195,78 +195,6 @@ void SetPlatformCrashKey() {
 #endif
 }
 
-std::string GetCrashValueForGinWrappable(gin::WrapperInfo* info) {
```

Comment:
Can you also remove the api class headers related to this block.

Reply:
Oh yes, thanks @deepak1556! Taking those out now 🙇‍♀️

Reply:
Looks like one addition was made that depends on something in the old headers, finding that specific class header and popping it in. 🙂